definitions/npm/fs-extra_v7.x.x/flow_v0.104.x-/fs-extra_v7.x.x.js

```diff
@@ -1,773 +1,832 @@
 declare module "fs-extra" {
   import type { Stats, ReadStream, WriteStream } from "fs";
+  import typeof fsTypes from "fs";
 
-  declare export type SymlinkType = "dir" | "file";
-  declare export type FsSymlinkType = "dir" | "file" | "junction";
+  declare type SymlinkType = "dir" | "file";
+  declare type FsSymlinkType = "dir" | "file" | "junction";
 
-  declare export type CopyFilterSync = (src: string, dest: string) => boolean;
-  declare export type CopyFilterAsync = (
+  declare type CopyFilterSync = (src: string, dest: string) => boolean;
+  declare type CopyFilterAsync = (
     src: string,
     dest: string
   ) => Promise<boolean>;
 
-  declare export type CopyOptions = {
+  declare type CopyOptions = {
     dereference?: boolean,
     overwrite?: boolean,
     preserveTimestamps?: boolean,
     errorOnExist?: boolean,
-    recursive?: boolean,
-    ...
+    recursive?: boolean
```

Comment:
Please don't remove all those inexact notations `...`, they are intended. Or convert those type to exact.

Reply:
Thank you! I fixed it locally at my place a long time ago, but there was no time to make a pr. I was sure that I had the relevant version of the source and that nobody had had time to make any commits here yet. Last commit 1 August 2019. I fixed it about September 2019, but apparently I already had an out-of-date version. I rolled it back.

Reply:
Okay yes, that's a bit weird to force ES modules for a node lib like this one.

Thanks for the PR!